authentication-guides/jira/README.md

```diff
@@ -0,0 +1,267 @@
+---
+name: Jira Approvals   # The name of the guide
+description: Send Proactive Jira Approval Notifications   # Brief description of the guide
+time_in_minutes: 60   # Time in minutes to complete the guide
+difficulty_level: Intermediate  # Difficulty level: "Beginner", "Intermediate", or "Advanced"
+---
+
+# Use Case Guide: JIRA Approvals
+
+Author: @Neal Moran
+Last edited time: February 28, 2024 1:03 PM
+Workspace: Events
+iPaaS: No iPaaS
+Technologies: JIRA
+Difficulty level: Intermediate
+Estimated time: 30 mins
+Status: Pending Publishing
+
+### Subtitle
+
+<aside>
+✨ JIRA Approval Notifications
+
+</aside>
+
+# Introduction
+
+[JIRA Automation](https://www.atlassian.com/software/jira/features/automation) is a feature that allows you to build automations and workflows based on events that occur within JIRA Service Desk.
+
+In this tutorial, we will explore how to use Creator Studio to notify approvers in-bot in near-real time when a new approval is pending.
+
+Let's get started!
+
+# Prerequisites
+
+<aside>
```

Comment:
aside isn't supported in HTML

@sswiderek-moveworks we should remove from our notion template to make the import more seamless